webrender/src/tiling.rs

```diff
@@ -456,12 +474,17 @@ impl AlphaBatcher {
     }
 }
 
-struct RenderTargetContext<'a> {
+struct CompileTileContext<'a> {
     layer_store: &'a Vec<StackingContext>,
     prim_store: &'a PrimitiveStore,
     render_task_id_counter: AtomicUsize,
 }
 
+struct RenderTargetContext<'a> {
+    layer_store: &'a Vec<StackingContext>,
```

Comment:
Any reason not to use `&'a [StackingContext]`? I think that would reduce the indirections on access